lab1.asm

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
Looks good. Before we merge, move your work to the file `q_one/q_one.asm` (located [here](https://github.com/NekesaH/AssemblyLab1/tree/q-one-part-one/q_one)) for consistency in our folder structure. Then remove this current file `lab1.asm`